build/msvc/build.cmd

```diff
@@ -204,103 +213,109 @@ IF NOT EXIST vcDlls (
+	cmake --build . --config %Configuration%
 	IF errorlevel 1 goto error_end
-	%CMAKE_DIR%\bin\cmake.exe --build . --config %Configuration% --target install
+	cmake --build . --config %Configuration% --target install
 	IF errorlevel 1 goto error_end
     
     popd
 )
 
 
 IF NOT EXIST %TEMP_DIR%\lpng170b75.zip (
-	%CYGWIN_DIR%\bin\wget.exe -O %TEMP_DIR%/lpng170b75.zip http://sourceforge.net/projects/libpng/files/libpng17/1.7.0beta75/lp170b75.zip/download
-    IF errorlevel 1	goto error_end
+	%CYGWIN_DIR%\bin\wget.exe -O %TEMP_DIR%/lpng170b75.zip ^
+			http://sourceforge.net/projects/libpng/files/libpng17/1.7.0beta75/lp170b75.zip/download
+				IF errorlevel 1	goto error_end
 )
 IF NOT EXIST lp170b75 (
-	%CYGWIN_DIR%\bin\unzip.exe -q %TEMP_DIR%/lpng170b75.zip
+	cmake -E tar %TEMP_DIR%/lpng170b75.zip
 	pushd lp170b75
-	%CMAKE_DIR%\bin\cmake.exe -G "%VS_CMAKE%" . -DCMAKE_INSTALL_PREFIX=..\%INSTALL_DIR%
+	cmake -G "%VS_CMAKE%" . -DCMAKE_INSTALL_PREFIX=..\%INSTALL_DIR%
 	IF errorlevel 1	goto error_end
-	%CMAKE_DIR%\bin\cmake.exe --build . --config %Configuration% --target install
+	cmake --build . --config %Configuration% --target install
 	IF errorlevel 1	goto error_end
 	popd
 )
 
 IF NOT EXIST %TEMP_DIR%\expat-2.1.0.tar (
-	%CYGWIN_DIR%\bin\wget.exe -O %TEMP_DIR%/expat-2.1.0.tar.gz http://sourceforge.net/projects/expat/files/expat/2.1.0/expat-2.1.0.tar.gz/download
-	%CYGWIN_DIR%\bin\gzip.exe -d %TEMP_DIR%/expat-2.1.0.tar.gz
+	%CYGWIN_DIR%\bin\wget.exe -O %TEMP_DIR%/expat-2.1.0.tar.gz ^
+			http://sourceforge.net/projects/expat/files/expat/2.1.0/expat-2.1.0.tar.gz/download
+	cmake -E tar %TEMP_DIR%/expat-2.1.0.tar.gz
 )
 IF NOT EXIST expat-2.1.0 (
-	%CYGWIN_DIR%\bin\tar.exe -xf %TEMP_DIR%/expat-2.1.0.tar
+	cmake -E tar %TEMP_DIR%/expat-2.1.0.tar
 )
 
 IF NOT EXIST expat-2.1.0.build (
     mkdir expat-2.1.0.build
     
     pushd expat-2.1.0.build
-	%CMAKE_DIR%\bin\cmake.exe -G "%VS_CMAKE%" -DCMAKE_INSTALL_PREFIX=..\%INSTALL_DIR% ..\expat-2.1.0
+	cmake -G "%VS_CMAKE%" -DCMAKE_INSTALL_PREFIX=..\%INSTALL_DIR% ..\expat-2.1.0
 	IF errorlevel 1 goto error_end
-	%CMAKE_DIR%\bin\cmake.exe --build . --config %Configuration%
+	cmake --build . --config %Configuration%
 	IF errorlevel 1 goto error_end
-	%CMAKE_DIR%\bin\cmake.exe --build . --config %Configuration% --target install
+	cmake --build . --config %Configuration% --target install
 	IF errorlevel 1 goto error_end
 	popd
 )
 
 IF NOT EXIST exiv2-%EXIV2_COMMIT% (
-    %CYGWIN_DIR%\bin\svn.exe co -r %EXIV2_COMMIT% svn://dev.exiv2.org/svn/trunk exiv2-%EXIV2_COMMIT%
+    svn co -r %EXIV2_COMMIT% svn://dev.exiv2.org/svn/trunk exiv2-%EXIV2_COMMIT%
 )
 
 IF NOT EXIST exiv2-%EXIV2_COMMIT%.build (
 	mkdir exiv2-%EXIV2_COMMIT%.build
 
     pushd exiv2-%EXIV2_COMMIT%
     SET EXIV2_CMAKE=
-	%CMAKE_DIR%\bin\cmake.exe -G "%VS_CMAKE%" -DCMAKE_PROGRAM_PATH=%SVN_DIR%  -DCMAKE_INSTALL_PREFIX=..\%INSTALL_DIR% -DEXIV2_ENABLE_BUILD_SAMPLES=OFF -DEXIV2_ENABLE_CURL=OFF -DEXIV2_ENABLE_SSH=OFF
```

Comment:
Exiv uses/used that to determine the actual version during compilation. With that you could query the version using exiv2 api. Now with git this might no longer work..